src/olympia/amo/tests/__init__.py

```diff
@@ -228,6 +228,9 @@ def create_sample(name=None, **kw):
     if not created:
         sample.__dict__.update(kw)
         sample.save()
+
+    sample.flush()
```

Comment:
Shouldn't the cache be flushed when saving ? Same for the other 2 ?

Reply:
I think this may even be old code, I went back and forth on this quite a lot so I may have missed it. I'll double-check and will probably remove it.

django-waffle does indeed call `.flush()` on save and it shouldn't be necessary for the create case.